rfcs/hse/0008-rework-cn-stats.md

```diff
@@ -0,0 +1,182 @@
+# Rework Internal CN Tree Statistics
+
+## Problem
+
+Internal CN tree statistics are confusing, redundant, misleading and inefficient.
+
+1. There is much duplication caused by the same statistic appearing in different
+   structures.  For example, `key_stats.tot_vlen`, `kblk_metrics.tot_val_bytes`,
+   and `kvset_metrics.tot_val_bytes` all track the same statistic.
+1. `kvset_metric` and `kvset_stats` are redundant. We should pick one.
+1. Many metrics don't measure what their name implies.  See *Misleading statistics*.
```

Comment:
You reference the section like https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown#16426829